### ISSUE TITLE
About the time cost of lora fine-tuning

### ISSUE THREAD

**Issue by cjfcsjt** [tgxs002/align_sd#12]
Thanks for your great work!
I wonder how much time it costs to train the Lora on the dataset (1M DiffusionDB + subset of Laion5B) on, for example, 4 GPUs?

**Comment by tgxs002** [comment]
It takes a few hours to train 10k iterations with batch size 40. The exact time depends on the type of GPU.

**Comment by cjfcsjt** [comment]
Thanks for your reply!! 
I have another question about the training dataset: According to the paper, it utilizes **{20,205 prompts, 79,167 images}** to _train the classifier_; while utilizing **37,572 preferred generated text-image pairs** + **21,108 non-preferred text-image pairs** (from DiffusionDB 1M) + **The 200,231 regularization text-image pairs** (from 625k subset of LAION-5B) to _fine-tune the Lora_. Is my understanding correct?
I wonder if it is possible to directly fine-tune the Lora using the **{20,205 prompts, 79,167 images}** human preference dataset?

**Comment by tgxs002** [comment]
Yes, the statistics are correct. You can try tuning the LoRA on HPD, but I feel the effect may be limited by the dataset size. The noise of the dataset may also be an issue.
